src/model/kubernetes.js

```diff
@@ -0,0 +1,329 @@
+                  ls`,
+                ],
+                volumeMounts: [
+                  {
+                    name: 'data',
+                    mountPath: '/data',
+                  },
+                  {
+                    name: 'credentials',
+                    mountPath: '/credentials',
+                    readOnly: true,
+                  },
+                ],
+                env: [
+                  {
+                    name: 'GITHUB_SHA',
+                    value: this.buildId,
+                  },
+                ],
+              },
+            ],
+            containers: [
+              {
+                name: 'main',
+                image: `${this.baseImage.toString()}`,
+                command: [
+                  'bin/bash',
+                  '-c',
+                  `for f in ./credentials/*; do export $(basename $f)="$(cat $f)"; done
+                  cp -r /data/builder/action/default-build-script /UnityBuilderAction
+                  cp -r /data/builder/action/entrypoint.sh /entrypoint.sh
+                  cp -r /data/builder/action/steps /steps
+                  chmod -R +x /entrypoint.sh;
+                  chmod -R +x /steps;
+                  ./entrypoint.sh;
+                  `,
+                ],
+                env: [
+                  {
+                    name: 'GITHUB_WORKSPACE',
+                    value: '/data/repo',
+                  },
+                  {
+                    name: 'PROJECT_PATH',
+                    value: this.buildParameters.projectPath,
+                  },
+                  {
+                    name: 'BUILD_PATH',
+                    value: this.buildParameters.buildPath,
+                  },
+                  {
+                    name: 'BUILD_FILE',
+                    value: this.buildParameters.buildFile,
+                  },
+                  {
+                    name: 'BUILD_NAME',
+                    value: this.buildParameters.buildName,
+                  },
+                  {
+                    name: 'BUILD_METHOD',
+                    value: this.buildParameters.buildMethod,
+                  },
+                  {
+                    name: 'CUSTOM_PARAMETERS',
+                    value: this.buildParameters.customParameters,
+                  },
+                  {
+                    name: 'BUILD_TARGET',
+                    value: this.buildParameters.platform,
+                  },
+                  {
+                    name: 'ANDROID_VERSION_CODE',
+                    value: this.buildParameters.androidVersionCode.toString(),
+                  },
+                  {
+                    name: 'ANDROID_KEYSTORE_NAME',
+                    value: this.buildParameters.androidKeystoreName,
+                  },
+                  {
+                    name: 'ANDROID_KEYALIAS_NAME',
+                    value: this.buildParameters.androidKeyaliasName,
+                  },
```

Comment:
Yes! and now we have to over time find a way to use this kind of configuration file to schedule a docker file locally, maybe even on k3s or something similar (super out of scope, just sharing a thought).